examples/hello_json/hello_json.zig

```diff
@@ -7,8 +7,7 @@ const User = struct {
 };
 
 fn on_request(r: zap.Request) void {
-    if (!std.mem.eql(u8, r.method.?, "GET"))
-        return;
+    if (r.method == null or r.method.? == .GET) return;
```

Comment:
this should be != .GET

I am not sure if I like having to check 2 things instead of one

Reply:
> this should be != .GET
yes, that is a bug

> I am not sure if I like having to check 2 things instead of one
`r.method.?` should be checked anyways, I think